compute_endpoint/tests/unit/test_cli_behavior.py

```diff
@@ -438,12 +440,36 @@ def test_self_diagnostic(
     res = run_line("self-diagnostic")
     stdout = res.stdout_bytes.decode("utf-8")
 
-    assert stdout.count("== Diagnostic") >= 16
+    assert stdout.count("== Diagnostic") >= 17
```

Comment:
This seems somewhat prone to mistypes or updates.  I wonder if there's an easy way to single out self_diagnostic.py::commands and doing assert stdout.count(...) >= len(commands).   Feel like a predefined dictionary of named items is easier for expansion later.

But that's changing the layout and logic of the original PR, definitely out of scope (if even desired) for now.  This is just a comment for your thoughts.

Reply:
I like the idea and might follow up with something in a subsequent PR.